util/dvsim/CfgFactory.py

```diff
@@ -0,0 +1,103 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging as log
+import sys
+
+from CfgJson import load_hjson
+
+import FpvCfg
+import LintCfg
+import SimCfg
+import SynCfg
+
+
+def _load_config(path, initial_values):
```

Comment:
nit: We are referring to HJson files as `cfgs` - even on dvsim command line args. Perhaps we should consolidate (either use `cfgs` everywhere or `configs`).